crates/red_knot_python_semantic/src/types.rs

```diff
@@ -236,7 +236,7 @@ impl<'db> ClassType<'db> {
 #[salsa::interned]
 pub struct UnionType<'db> {
     /// the union type includes values in any of these types
-    elements: FxOrderSet<Type<'db>>,
+    pub elements: FxOrderSet<Type<'db>>,
 }
```

Comment:
Nit: I would prefer keeping the field private and instead add a `.contains` method to `UnionType` itself.